src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/RazorWorkspaceListenerInitializer.cs

```diff
@@ -14,7 +14,7 @@ internal sealed class RazorWorkspaceListenerInitializer
 {
     // This should be moved to the Razor side once things are announced, so defaults are all in one
     // place, in case things ever need to change
-    private const string _projectRazorJsonFileName = "project.razor.vscode.json";
+    private const string _projectRazorJsonFileName = "project.razor.vscode.bin";
```

Comment:
Would be good to move this to the razor side entirely (and have Roslyn consume it via an input param or option) as mentioned in the comment.  But no need to do now if this needs to go in urgently.